nix/pkgs/status-go/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, callPackage }:
+
+let
+  inherit (lib) getEnvWithDefault attrValues mapAttrs;
+
+  # Metadata common to all builds of status-go
+  meta = {
+    description = "The Status Go module that consumes go-ethereum.";
+    license = lib.licenses.mpl20;
+    platforms = with lib.platforms; linux ++ darwin;
+  };
+
+  # Source can be changed with a local override from config
+  source = callPackage ./source.nix { };
+
+  # Params to be set at build time, important for About section and metrics
+  goBuildParams = {
+    GitCommit = source.rev;
+    Version = source.cleanVersion;
+    # FIXME: This should be moved to status-go config.
+    IpfsGatewayURL = getEnvWithDefault "STATUS_GO_IPFS_GATEWAY_URL" "https://ipfs.status.im/";
```

Comment:
Use of `getEnvWithDefault` which uses `builtins.getEnv` will not work with Flake builds.